packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -57,12 +57,6 @@ export const getById = async (id: string): Promise<Sync | null> => {
 };
 
 export const createSync = async (nangoConnectionId: number, syncConfig: DBSyncConfig): Promise<Sync | null> => {
-    const existingSync = await getSyncByIdAndName(nangoConnectionId, syncConfig.sync_name);
-
-    if (existingSync || !syncConfig.id) {
-        return null;
-    }
-
```

Comment:
createSync is only used in the 2 cases above. Checking for an existing sync has been moved outside of the function